app/services/katello/candlepin_event_listener.rb

```diff
@@ -1,4 +1,5 @@
 module Katello
+  # TODO: Move this class to app/lib/katello/event_daemon/services with other service definitions
```

Comment:
do you still intend to do this?

Reply:
I think it's out of scope of this PR so that's a TODO for some future dev (maybe me)